added case study of data collection for rural water systems in Rwanda (#77)
opengisch/QField-docs: en/case-studies/rwanda-rural-water.rst

Data collection by QGIS/QField for O&M work of rural water supply systems in Rwanda
=========================================================================================================

By Rural Water and Sanitation Services Department(RWSS), Water and Sanitation Corporation (`WASAC <https://wasac.rw>`__), Rwanda

Goal
----

To conduct data collection of all rural water supply network in Rwanda, 
and keep updating the data continuously in order to improve operation & 
maintenance of waterworks.

Project preparation
-------------------

Before starting our data collection, we conducted the following things.
 - Develop our own PostGIS database

>  - Develop QGIS project template with Geopackage. The Geopackage table design should be equal to PostGIS. So we can just copy and paste to PostGIS.

Reviewer: ```suggestion
 - Develop QGIS project template with Geopackage. The Geopackage table design is equal to PostGIS to be able to copy and paste to PostGIS.
```

> Apart from preparing Android devices, if possible, you may purchase GPS devices. In WASAC, we bought Garmin GPSMAP 64S. 

Reviewer: ```suggestion
Apart from preparing Android devices, we purchased GPS devices for higher positioning accuracy. In WASAC, we bought Garmin GPSMAP 64S. 
```

Sometimes, GPS of smartphone and tablet is not very accurate, so we normally capture the same location by using Garmin GPS, then correct the location of QField's data after data collection work.

Data collection
---------------

Once we prepared Geopackage and QGIS project template, 
we conducted training of QGIS/QField in July 2018 and 
launched our data collection work in 27 districts in the whole country of Rwanda. 

> 27 engineers sent their Geopackage to central office in Kigali, 

Reviewer: ```suggestion
27 engineers sent their Geopackage to the central office in Kigali, 
```

> the MIS specialist validated and entered their data from Geopackage to PostGIS database. 

Reviewer: What is the "MIS specialist"? Has this been introduced before?

Author: It is a position in WASAC. MIS (Management Information System) Specialist. Let me add it.

Author: I added description of MIS in this commit(https://github.com/opengisch/QField-docs/pull/77/commits/5d15d0943a202e52235bb906d356519148aa2030).

We completed our initial data collection works until April 2019.

.. container:: clearer text-center

   .. image:: /images/rwanda-rural-water-1.png
      :width: 600px
      :alt: data collection procedure

Data distribution and updating
------------------------------

The most significant thing after data collection is ``updating``. 

> We have seen many organization in Africa, they normally failed to keep updating. 

Reviewer: ```suggestion
We have seen many organization in Africa, which failed to keep data up to date.
```

Several years later, their data will normally become too old, 
and most of them need to put efforts on data collection again. 

> However, WASAC is continuously updating all of the data until now. 

Reviewer: ```suggestion
WASAC decided to continuously update all of the data and keeps doing this until now. 
```

> We think QField works very well for this purpose.

Reviewer: ```suggestion
QField has proven to be very well suited for this purpose.
```

In order to distribute and updating the data, 
we developed a python script `postgis2qfield <https://github.com/WASAC/postgis2qfield>`__. 

> This ``postgis2qfield`` tool can extract the data from PostGIS and create Geopackages for each districts in Rwanda. 

Reviewer: ```suggestion
This ``postgis2qfield`` tool can extract the data from PostGIS and create Geopackages for each district in Rwanda. 
```

We upload these 27 geopackage together with QGIS project template to Google Drive storage. 
After that, those engineers in districts download their geopakage to Android device to continue adding and updating the data. 

> Once they completed updating, send geopackage to central office again, MIS specialist update PostGIS database and regenerate geopackages for QField.

Reviewer: ```suggestion
Once they completed updating, they sent the geopackage to central office again, MIS specialist update PostGIS database and regenerate geopackages for QField.
```

.. container:: clearer text-center

   .. image:: /images/rwanda-rural-water-2.png
      :width: 600px
      :alt: data distribution and updating procedure

Why didn't we use ``QFieldSync`` plugin for this purpose? 

Although QField has a nice QFieldSync plugin for linking between PostGIS and Geopackage, 

> we didn't select QFieldSync. Because QFieldSync plugin only can create a Geopackage only, 

Reviewer: That's actually not completely true, QFieldSync can prepare a subset of the data only. But it's always a bounding box / rectangular.

Reviewer: I wonder if we could just remove this paragraph? It highlights drawbacks of the QFieldSync plugin. One the one hand it's not completely correct (see above) and they will likely be inappropriate in the near future and start to be misleading for people reading this.

Author: > But it's always a bounding box / rectangular

Sure, I remember QFieldSync has this function. However, in WASAC, they need to separate data exactly by the boundary of district. Also, it is not easy for them to repeat the same process for all the districts. That is why I developed customized python script.

> I wonder if we could just remove this paragraph? 

Yeah, I think I can remove this paragraph.

Author: I deleted the paragraph in this commit(https://github.com/opengisch/QField-docs/pull/77/commits/088ffee04cd7ee0a23b4d85e222b66ddb00452eb).
